design/comments.go

```diff
@@ -131,6 +131,21 @@ var _ = a.Resource("comments", func() {
 		a.Response(d.NotFound, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
+	a.Action("delete", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.DELETE("/:id"),
```

Comment:
Yes, it is. See here: https://github.com/almighty/almighty-core/blob/master/design/comments.go#L112